### PR TITLE
Bugfix/1339 conformance spark job fails with npe

### LINKED ISSUE #1339: Conformance spark job fails with a NPE
## Describe the bug
For the current version of enceladus in `develop` (5d5180d2), the conformance spark job fails with a NPE:
```
java.lang.NullPointerException
	at za.co.absa.atum.core.Accumulator.getControlMeasure(Accumulator.scala:67)
	at za.co.absa.atum.core.ControlFrameworkState.getControlMeasure(ControlFrameworkState.scala:155)
	at za.co.absa.atum.core.Atum$.getControlMeasure(Atum.scala:143)
	at za.co.absa.enceladus.conformance.DynamicConformanceJob$.main(DynamicConformanceJob.scala:120)
	at za.co.absa.enceladus.conformance.DynamicConformanceJob.main(DynamicConformanceJob.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
	at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:845)
	at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:161)
	at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:184)
	at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:86)
	at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:920)
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:929)
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
```

## To Reproduce
Steps to reproduce the behavior OR commands run:
Run a conformance job.

## Expected behaviour
The job is expected to finish successfully and produce output.

## Desktop (please complete the following information):
 - OS: Win 10
 - Version enceladus 2.5.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by AdrianOlosutean**
Fixes #1339 by providing the path to the INFO file in the conformance job

### CODE REVIEW

File: spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala [hunk 1]
```diff
@@ -205,14 +205,18 @@ object DynamicConformanceJob {
     newVersion
   }
 
-  private def initFunctionalExtensions(reportVersion: Int)(implicit spark: SparkSession, dao: MenasDAO, cmd: ConfCmdConfig): Unit = {
+  private def initFunctionalExtensions(reportVersion: Int)(implicit spark: SparkSession,
```
Comment (benedeki) [inline spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala:208]: I would just make it a standard parameter, same as in **Standardization**. IMHO no need to be `implicit`

Comment (yruslan) [reply spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala]: Now `cmd: ConfCmdConfig` is implicit, but `pathCfg: PathCfg` is not...

File: spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala [hunk 2]
```diff
@@ -112,7 +112,6 @@ object DynamicConformanceJob {
 
       log.info("Conformance finished successfully")
     } finally {
-      Atum.getControlMeasure.runUniqueId
```
Comment (benedeki) [inline spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala:115]: Just curious, what was the point of this call?

Comment (AdrianOlosutean) [reply spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala:115]: Maybe leftover code. The problem with this line was that it was hiding the proper exception which was had the message ControlFramework tracking not initialized

**Pull request merged**